recipes/yggdrasil/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler:
```

Comment:
This file shouldn't be needed now the compiler migration is complete.

Reply:
I have removed the conda_build_config.yaml